### DISCUSSION TITLE
Claim Settings

### DISCUSSION THREAD

**Discussion by LuckyFish312** [GriefPrevention/GriefPrevention#607] [Ideas]
As a player,
You should be able to have a claim settings (like the island settings in skyblock),
Where you can enable/disable if players on your claim can do certain things,
Such as the ability to use anvils or enderpearls on your claim,
Or if they can shear sheep and eat cake,
Or use repeaters and noteblocks and chorus fruit.
Because I think it would make Grief Prevention much better,
And make it so other players can do a lot more on your claim,
And it would be much easier than using a bunch of /trusts like /ContainerTrust and /AccessTrust.

Below I have some pictures of an example, except there should be more options than in the picture, such as if they can eat cake and milk cows etc.
![Screen Shot 2019-09-04 at 7 22 31 AM](https://user-images.githubusercontent.com/54894769/64255413-f1709d00-cee6-11e9-81f7-1ef71a01b3a4.png)
![Screen Shot 2019-09-04 at 7 22 35 AM](https://user-images.githubusercontent.com/54894769/64255435-fc2b3200-cee6-11e9-86f6-bcc6ca6d3b72.png)
![Screen Shot 2019-09-04 at 7 25 52 AM](https://user-images.githubusercontent.com/54894769/64255446-ff262280-cee6-11e9-8231-7ac39fa14c19.png)
![Screen Shot 2019-09-04 at 7 26 01 AM](https://user-images.githubusercontent.com/54894769/64255452-02b9a980-cee7-11e9-88eb-1bd24abd4ae4.png)

**Comment by RoboMWM** [comment]
Sounds like GriefPrevention flags

**Comment by LuckyFish312** [comment]
What's that?

**Comment by RoboMWM** [comment]
An addon for GP
